php/README.md

```diff
@@ -45,27 +45,26 @@ The following table lists the configurable parameters of the PHP chart and their
 |  `sharedPath` | Path shared by busybox/nginx/fpm | `/var/www/html` |
 |  `replicaCount` | Number of pods to start with deployment | `1` |
 |  `strategy` | Update strategy of pod in deployment | `type`: `RollingUpdate` |
-|  `podAnnotation` | Annotation specified for pod in deployment | {} |
+|  `podAnnotation` | Annotation specified for pod in deployment | `{}` |
 |  `imagePullSecrets` | Name of Secret resource containing private registry credentials | `[]` |
 |  `extraVolumes` | Additional volumes to all container | `[]` |
 |  `extraVolumeMounts` | Additional volumeMounts to all container | `[]` |
 |  `tolerations` | Pod taint tolerations for deployment | `[]` |
 |  `affinity` | Node / Pod affinities | `{}` |
 |  `service.type` | Changes to ClusterIP automatically if ingress enabled | `LoadBalancer` |
 |  `service.port` | Port to advertise the main web service in LoadBalancer mode | `8080` |
```

Comment:
@cw-niinuma 
I thought the default value of `service.port` is nil because that was not described in `values.yaml`.